com.unity.cinemachine/Editor/Menus/CinemachineMenu.cs

```diff
@@ -247,6 +250,8 @@ public static LensSettings MatchSceneViewCamera(Transform sceneObject)
                 }
             }
             return lens;
+
+            static bool AtOrigin(Vector3 a) => a.x == 0f && a.y == 0f && a.z == 0f;
```

Comment:
this is a rather complicated way of saying sceneObject.transform.position == Vector3.zero